com.gluonhq.scenebuilder.yml

```diff
@@ -0,0 +1,48 @@
+app-id: com.gluonhq.scenebuilder
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: SceneBuilder
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+cleanup:
+  - /share/info/*
+  - /share/locale/*
+  - /share/man/*
+  - /share/runtime/locale/*
+  - 
+modules:
+  - name: SceneBuilder
+    buildsystem: simple
+    only-arches:
+    build-commands:
+    - mkdir -p deb-package
+    - ar p SceneBuilder-18.0.0.deb data.tar.xz | tar -xJf - -C deb-package
```

Comment:
Should be changed to `SceneBuilder.deb` after `dest-filename` is used to remove version.